account_payment_partner/wizard/payment_order_create.py

```diff
@@ -28,10 +28,10 @@ class PaymentOrderCreate(models.TransientModel):
 
     @api.model
     def extend_payment_order_domain(self, payment_order, domain):
-        res = super(PaymentOrderCreate, self).extend_payment_order_domain(
+        super(PaymentOrderCreate, self).extend_payment_order_domain(
             payment_order, domain)
         domain += ['|', '|',
                    ('invoice', '=', False),
                    ('invoice.payment_mode_id', '=', False),
                    ('invoice.payment_mode_id', '=', payment_order.mode.id)]
-        return res
+        return True
```

Comment:
@alexis-via's change in return True mimicks the behaviour of the super method.
Actually, it's the return in the super method that should be removed as that method has no meaningful return value.